### DISCUSSION TITLE
UserWarning: You requested to overfit but enabled val/test dataloader shuffling. We are turning it off for you.

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Lightning-AI/pytorch-lightning#11855] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I am getting this warning "UserWarning: You requested to overfit but enabled val/test dataloader shuffling. We are turning it off for you." though shuffle=False. Please check code https://github.com/PyTorchLightning/pytorch-lightning/discussions/11851

**Comment by rohitgr7** [comment]
this warning is removed from master, and will not come up during the next release.
